CMakeLists.txt

```diff
@@ -101,6 +97,39 @@ IF (CMAKE_COMPILER_IS_GNUCC)
 	ENDIF ( ${CMAKE_BUILD_TYPE} MATCHES "Debug" )
 ENDIF (CMAKE_COMPILER_IS_GNUCC)
 
+
+# Set some specific settings for Mac OS X - Darwin
+#
+IF (MACOSX)
+    
+    # For MAC OS X we set the GUI to ON for ssconfig etc.
+    SET(SC_GLOBAL_GUI ON)
```

Comment:
That is a configuration flag that should be set by the user independently of the target platform. Even on a Mac you can build without GUI if you want. I don't see a reason to remove that flexibility.